packages/core/src/components/AlertBanner/AlertBannerText/AlertBannerText.tsx

```diff
@@ -23,12 +20,7 @@ const AlertBannerText: FC<AlertBannerTextProps> = ({ text, marginLeft = false, i
   const isOverflowing = useIsOverflowing({ ref: componentRef });
 
   return (
-    <Tooltip
-      position="bottom"
-      content={isOverflowing && text}
-      showTrigger={TOOLTIP_SHOW_TRIGGERS}
-      hideTrigger={TOOLTIP_HIDE_TRIGGERS}
-    >
+    <Tooltip position="bottom" content={isOverflowing && text} showTrigger="mouseenter" hideTrigger="mouseleave">
```

Comment:
Is it intentional that `showTrigger` and `hideTrigger` were changed from array to string?

Reply:
No lol